pkg/database/scopes.go

```diff
@@ -69,6 +69,17 @@ func WithAuditSearch(from, to string) Scope {
 	}
 }
 
+// WithAuditRealmID returns a scope that adds querying for Audit events by realm.
+func WithAuditRealmID(r string) Scope {
```

Comment:
I'm not a huge fan of this accepting a string. I'd rather accept an interface{} or uint.